devito/ir/ietxdsl/cluster_to_ssa.py

```diff
@@ -378,76 +337,6 @@ def match_and_rewrite(self, op: func.FuncOp, rewriter: PatternRewriter):
         ])
 
 
-class _DevitoStencilToStencilStencil(RewritePattern):
-    """
-    This converts the `devito.stencil` op into the following:
```

Comment:
I think we will need a similar docstring of the newly supported cases, in the added lines